releasenotes/notes/fix-logger-standard-logging-rate-envvar-155e113deea71e85.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Deprecate the (DD/DATADOG)_LOGGING_RATE_LIMIT variable in favour of the
```

```suggestion
    Deprecate the DD_LOGGING_RATE_LIMIT variable in favor of the
```

We can just refer to it as `DD_LOGGING_RATE_LIMIT`, while `get_env()` looks for both, `DATADOG_` prefix is actually the old way (years ago).